app/logic/Mail/IMAP/IMAPAccount.ts

```diff
@@ -375,6 +428,11 @@ export class IMAPAccount extends MailAccount {
   }
 }
 
+/** The three types of IMAP namespace. */
+type IMAPNamespace = "personal" | "other" | "shared";
```

Comment:
What is "other"? Public?

Reply:
No, "shared" is for public folders and "other" is for other users.

Reply:
Could you please document that in the JSDoc?